demo/startRedis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Starting redis-server as a UNIX daemon process and
+# putting the process to sleep state for 1 second
+redis-server --daemonize yes && sleep 1
+
+# Adding default hashlist to the redis-server
+redis-cli < /usr/local/hashlist.redis
+
+# Saving the added entries
+redis-cli save
+
+# closing the cli prompt
+redis-cli shutdown
```

Comment:
Is stopping redis-cli and restarting redis-server needed?  'Save' from above should be sufficient. Subsequently, is sleep really needed above?

Reply:
yes. Only adding redis-cli commands results in connection refused. So the redis-server commands are required